src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/UserDefinedImplicitConversions.cs

```diff
@@ -132,7 +132,7 @@ private static void ComputeUserDefinedImplicitConversionTypeSet(TypeSymbol s, Ty
         /// type to any target type.
         /// </summary>
         /// <remarks>
-        /// Currently allowAnyTarget flag is only set to true by AnalyzeImplicitUserDefinedConversionForSwitchGoverningType,
+        /// Currently allowAnyTarget flag is only set to true by AnalyzeImplicitUserDefinedConversionForV6SwitchGoverningType,
```

Comment:
Consider using standard cref syntax so that go to definition and rename work for this reference.